pkg/nsx/services/securitypolicy/store_test.go

```diff
@@ -180,90 +248,14 @@ func Test_queryRule(t *testing.T) {
 	patches2 := gomonkey.ApplyMethod(reflect.TypeOf(tc), "ConvertToGolang",
 		func(_ *bindings.TypeConverter, d data.DataValue, b bindings.BindingType) (interface{}, []error) {
 			mId, mTag, mScope := "11111", "11111", "11111"
-			m := model.Rule{
+			m := model.SecurityPolicy{
 				Id:   &mId,
 				Tags: []model.Tag{{Tag: &mTag, Scope: &mScope}},
 			}
-			var j interface{} = &m
+			var j interface{} = m
 			return j, nil
 		})
 	defer patches2.Reset()
 
-	patches3 := gomonkey.ApplyFunc(queryTagCondition, func(service *SecurityPolicyService) string {
-		return ""
-	})
-	defer patches3.Reset()
-
-	queryRule(service, &wg, fatalErrors)
-}
-
-func TestListSecurityPolicyID(t *testing.T) {
```

Comment:
As we discussed, service.QueryResource just for store initialization, we may still need a func to query list, just like TestListSecurityPolicyID, why we remove this test func? can we bring it back?

Reply:
ok